tests/test_connector.py

```diff
@@ -415,10 +418,10 @@ def _get_object(self, url, **opts):
         resp = requests.Response
         resp.status_code = 200
         if "_page_id" in url:
-            resp.content = jsonutils.dumps({"result": [6,7,8,9,10]})
+            resp.content = jsonutils.dumps({"result": [6, 7, 8, 9, 10]})
         else:
             resp.content = jsonutils.dumps(
-                               {"result": [1,2,3,4,5], "next_page_id": 1})
```

Comment:
same as above

Reply:
Yes, but then Flake8 throws an error because the line is too long. I will add a disable flag.